src/Cms/Language.php

```diff
@@ -176,18 +176,18 @@ public static function create(array $props): static
 
 		$language->save();
 
+		// update the main languages collection in the app instance
+		$kirby->languages(false)->append($language->code(), $language);
```

Comment:
`Question`: What's the reasoning behind moving this line to the top?

Reply:
That change wasn't necessary. I had a different version of the code before which needed to make sure that the language collection is updated first. But I just reverted it to avoid unnecessary changes and confusion here.